examples/data-drift-detection/data-drift.md

```diff
@@ -0,0 +1,179 @@
+ 
+In the dataset we have some columns that we don't need, which are `State`, `Country`, `Region`, `Month`, `Day`, `Year`. We don't need the location based columns besides `City`, as we will create our model on the data for the city of New Orleans. We also don't need the date based columns, as we will only use data for one city for which we can use indexes as our date guide. Thus we delete them.
+
+```python
+import pandas as pd
+
+# Your path to the dbt project.
+PATH_PREFIX = ''
+
+df = pd.read_csv(f'{PATH_PREFIX}/data/city_temperature.csv')
+
+df.pop('State')
+df.pop('Country')
+df.pop('Region')
+df.pop('Month')
+df.pop('Day')
+df.pop('Year')
+
+df.to_csv(f'{PATH_PREFIX}/data/city_temperature.csv')
+```
+
+Now that we have our modified CSV file, we `dbt seed` the dataset upstream. As we have our data upstream now, we can write the following dbt model in our project:
+
+```sql
+{{ config(materialized='table') }}
+
+WITH source_data as (
+    select * from {{ ref('city_temperature') }}
+)
+
+SELECT
+    AvgTemperature as y,
+    a as x
+FROM
+    source_data
+WHERE
+    City = 'New Orleans'
+```
+
+In our model, we only select the data for the city of New Orleans, and set `AvgTemperature` as `y`, as it is our metric value. We also set the `a` values in our data to `x` for indexing.
+
+## The script
+
+Now that we have our model to work on, we can write our script so that fal can show us it's magic. For detecting data drift, we use a test called the [Kolmogorov-Smirnov test](https://en.wikipedia.org/wiki/Kolmogorov–Smirnov_test), specifically the 2 sample test. What the test does is that it essentially finds the underlying distributions of each of the data and checks if they are the same with a confidence level. The confidence level is generally set to 95% for optimal results. For our script, we will use the `ks_2samp` function provided by the `scipy` library, which is the test we talked about. It takes two sets of data as its arguments, then returns the test statistic and the probability level, the `p_value`. We take and compare the `p_value` against `1 - confidance_level = 0.05`. If the p_value is less than it, data drift is present.
+
+```python
+# Your path to the dbt project.
+PATH_PREFIX = ''
+
+import sys
+sys.path.append(f'{PATH_PREFIX}/fal_scripts')
```

Comment:
why do you add this? I think all imports are from libraries? Also, now it is possible to import natively. use the fal-scripts-path variable if necessary: https://fal-docs-ekvqb18d4-features-and-labels.vercel.app/Reference/fal-scripts-path